diff -urN nam-1.9.orig/editview.cc nam-1.9/editview.cc
--- nam-1.9.orig/editview.cc	2003-02-26 21:52:52.000000000 +0100
+++ nam-1.9/editview.cc	2003-10-31 18:16:22.000000000 +0100
@@ -61,7 +61,7 @@
 	if (tk_!=0) {
 		Tcl& tcl = Tcl::instance();
 		cmd_ = Tcl_CreateCommand(tcl.interp(), Tk_PathName(tk_), 
-					 command, 
+(int (*)(void*, Tcl_Interp*, int, const char**))command, 
 					 (ClientData)this, DeleteCmdProc);
 	}
 	char str[256];
@@ -84,7 +84,7 @@
 	if (tk_!=0) {
 		Tcl& tcl = Tcl::instance();
 		cmd_ = Tcl_CreateCommand(tcl.interp(), Tk_PathName(tk_),
-		                         command, (ClientData) this,
+(int (*)(void*, Tcl_Interp*, int, const char**))command, (ClientData) this,
 		                         DeleteCmdProc);
 	}
 
diff -urN nam-1.9.orig/graphview.cc nam-1.9/graphview.cc
--- nam-1.9.orig/graphview.cc	2003-02-26 21:52:53.000000000 +0100
+++ nam-1.9/graphview.cc	2003-10-31 18:15:33.000000000 +0100
@@ -53,7 +53,7 @@
         : View(name, NONSQUARE, 30, 30), next_(NULL), graph_(g)
 {
   Tcl& tcl = Tcl::instance();
-  tcl.CreateCommand(Tk_PathName(View::tk()), command, (ClientData)this, 0);
+  tcl.CreateCommand(Tk_PathName(View::tk()), (int (*)(void*, Tcl_Interp*, int, const char**))command, (ClientData)this, 0);
 }
 
 // void GraphView::draw()
diff -urN nam-1.9.orig/main.cc nam-1.9/main.cc
--- nam-1.9.orig/main.cc	2003-02-26 21:52:54.000000000 +0100
+++ nam-1.9/main.cc	2003-10-31 18:14:39.000000000 +0100
@@ -545,8 +545,8 @@
 	}
 #endif
 
-	tcl.CreateCommand("adios", cmd_adios);
-	tcl.CreateCommand("version", cmd_version);
+	tcl.CreateCommand("adios", (int (*)(void*, Tcl_Interp*, int, const char**))cmd_adios);
+	tcl.CreateCommand("version", (int (*)(void*, Tcl_Interp*, int, const char**))cmd_version);
 
 #ifdef WIN32
 	platformInit(interp);
diff -urN nam-1.9.orig/netview.cc nam-1.9/netview.cc
--- nam-1.9.orig/netview.cc	2003-02-26 21:52:54.000000000 +0100
+++ nam-1.9/netview.cc	2003-10-31 18:14:08.000000000 +0100
@@ -76,7 +76,7 @@
 	if (tk_!=0) {
 		Tcl& tcl = Tcl::instance();
 		cmd_ = Tcl_CreateCommand(tcl.interp(), Tk_PathName(View::tk()),
-					 command, (ClientData)this, 
+(int (*)(void*, Tcl_Interp*, int, const char**))command, (ClientData)this, 
 					 DeleteCmdProc);
 	}
 }
diff -urN nam-1.9.orig/paint.cc nam-1.9/paint.cc
--- nam-1.9.orig/paint.cc	2003-02-26 21:52:54.000000000 +0100
+++ nam-1.9/paint.cc	2003-10-31 18:15:08.000000000 +0100
@@ -122,7 +122,7 @@
 	color.blue = b;
 
 	XColor *p = Tk_GetColorByValue(tk,&color);
-	return (Tk_NameOfColor(p));
+	return (char*)(Tk_NameOfColor(p));
 
 }
 int Paint::lookup(const char * color, int linewidth)
